vital/types/rotation.cxx

```diff
@@ -294,6 +294,20 @@ compose_rotations(
   return kwiver::vital::rotation_< T >(R);
 }
 
+template < typename T >
+rotation_< T >
+compose_rotations( rotation_< T > const & platform_rotation,
+                   rotation_< T > const & sensor_rotation )
+{
+  T platform_yaw, platform_pitch, platform_roll,
+    sensor_yaw,   sensor_pitch,   sensor_roll;
```

Comment:
```suggestion
  T platform_yaw, platform_pitch, platform_roll;
  T sensor_yaw,   sensor_pitch,   sensor_roll;
```

(As much as I'm not a fan of multiple declarations per line, I'm even less a fan of multiple declarations [of a single statement] on *multiple* lines. Particularly as not doing that is the exact same number of characters 🙂.)